<launch>

  <!-- Visualizer用のパラメータ -->
  <!-- <arg name="publish_ball" default="true" />   -->
  <!-- <arg name="publish_blue" default="true" />   -->
  <!-- <arg name="publish_yellow" default="true" /> -->

  <!-- grSimのロボットを動かす場合はtrue -->
  <arg name="sim" default="false" />

  <!-- ビジュアライザの起動 -->
  <include file="$(find consai2_examples)/launch/visualizer.launch">
    <!-- <arg name="publish_ball" default="$(arg publish_ball)" />     -->
    <!-- <arg name="publish_blue" default="$(arg publish_blue)" />     -->
    <!-- <arg name="publish_yellow" default="$(arg publish_yellow)" /> -->
  </include>

  <include file="$(find consai2_sender)/launch/sender.launch">
    <arg name="load_description" value="false" />
    <arg name="sim" value="$(arg sim)" />
  </include>

  <node name="consai2_control" pkg="consai2_control" type="example_control.py" required="true" output="screen" />

  <node name="game" pkg="consai2_game" type="game.py" required="true" output="screen" />
</launch>
